datacenterlight/templatetags/custom_tags.py

```diff
@@ -21,7 +21,6 @@ def change_lang(context, lang=None, *args, **kwargs):
 
     return "%s" % url
 
```

Comment:
By deleting this line, I think we are breaking PEP8. Here's what flake8 complains
```
./datacenterlight/templatetags/custom_tags.py:24:1: E302 expected 2 blank lines, found 1```